packages/flame/lib/src/events/messages/tap_cancel_event.dart

```diff
@@ -19,4 +19,7 @@ class TapCancelEvent extends Event {
   /// The id of the event that has been cancelled. This id corresponds to the
   /// id of the previous [TapDownEvent].
   final int pointerId;
+
+  @override
+  String toString() => 'TapCancelEvent(pointerId;$pointerId)';
```

Comment:
```suggestion
  String toString() => '$TapCancelEvent(pointerId;$pointerId)';
```
Will help in keeping class name in this string in-sync with the actual class name in future as well.